server/androidtv_server.py

```diff
@@ -0,0 +1,809 @@
+      "sources": ["https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/Introducing%20Google%20Nose.mp4"],
+      "card": "https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/Introducing%20Google%20Nose/card.jpg",
+      "background": "https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/Introducing%20Google%20Nose/bg.jpg",
+      "title": "Introducing Google Nose",
+      "studio": "Google"
+    }, {
+      "description": "Fusce id nisi turpis. Praesent viverra bibendum semper. Donec tristique, orci sed semper lacinia, quam erat rhoncus massa, non congue tellus est quis tellus. Sed mollis orci venenatis quam scelerisque accumsan. Curabitur a massa sit amet mi accumsan mollis sed et magna. Vivamus sed aliquam risus. Nulla eget dolor in elit facilisis mattis. Ut aliquet luctus lacus. Phasellus nec commodo erat. Praesent tempus id lectus ac scelerisque. Maecenas pretium cursus lectus id volutpat.",
+      "sources": ["https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/YouTube's%20ready%20to%20select%20a%20winner.mp4"],
+      "card": "https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/YouTube's%20ready%20to%20select%20a%20winner/card.jpg",
+      "background": "https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/YouTube's%20ready%20to%20select%20a%20winner/bg.jpg",
+      "title": "YouTube's ready to select a winner",
+      "studio": "Google"
+    }, {
+      "description": "Fusce id nisi turpis. Praesent viverra bibendum semper. Donec tristique, orci sed semper lacinia, quam erat rhoncus massa, non congue tellus est quis tellus. Sed mollis orci venenatis quam scelerisque accumsan. Curabitur a massa sit amet mi accumsan mollis sed et magna. Vivamus sed aliquam risus. Nulla eget dolor in elit facilisis mattis. Ut aliquet luctus lacus. Phasellus nec commodo erat. Praesent tempus id lectus ac scelerisque. Maecenas pretium cursus lectus id volutpat.",
+      "sources": ["https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/Levity%20Algorithm.mp4"],
+      "card": "https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/Levity%20Algorithm/card.jpg",
+      "background": "https://storage.googleapis.com/android-tv/Sample%20videos/April%20Fool's%202013/Levity%20Algorithm/bg.jpg",
+      "title": "Levity Algorithm",
+      "studio": "Google"
+    }]
+  }]
+
+"""
+
+
+# handler to get all videos in same category
+# the category information will be passed from the url
+class GetVideosInSameCategory(webapp2.RequestHandler):
+    def get(self):
+        # add artificial random latency
+        time.sleep(generate_random_latency())
+
+        category = self.request.get('category')
+        # we will randomize the database every time when the request is received by server
+        if random.random() > 0.5:
```

Comment:
can we remove this random stuff?  for testing, use another request url e.g. 

shuffle_category?name=xyz
dup_category?from=xyz&to=abc

Reply:
Yes, it makes sense that avoid random factor from the server side. 

Currently, the server has different urls to duplicate/ shuffle the channel, client can hit different end point randomly.